code/modules/clothing/spacesuits/void/zaddat_yw.dm

```diff
@@ -0,0 +1,207 @@
+/obj/item/clothing/suit/space/void/zaddat/engineer
+	name = "\improper Engineer's Guild Shroud"
+	desc = "This rugged Shroud was created by the Xozi Engineering Guild."
+	armor = list(melee = 10, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 100, rad = 100)
```

Comment:
```suggestion
	armor = list(melee = 0, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 100, rad = 100)
```

Reply:
The reason i put 10 armor is because of the usual item throwing that happens around, helping them not suffer much on passing items

Reply:
But its not really relevant